<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>diem_trungbinh</class>
 <widget class="QWidget" name="diem_trungbinh">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1190</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="BangDiem_Widget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>791</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_malop">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>185</y>
     <width>771</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lớp: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>651</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>BẢNG THỐNG KÊ ĐIỂM TRUNG BÌNH KHÓA HỌC</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>811</width>
     <height>681</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_TraDiemTB">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>89</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Tra Bảng Điểm</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>59</y>
     <width>561</width>
     <height>68</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLineEdit" name="lineEdit__MaLop">
      <property name="placeholderText">
       <string>Chữ cái</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Mã lớp</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>tableView</zorder>
  <zorder>BangDiem_Widget</zorder>
  <zorder>label_malop</zorder>
  <zorder>label_5</zorder>
  <zorder>pushButton_TraDiemTB</zorder>
  <zorder>gridLayoutWidget_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
